lib/parallel_cucumber/worker.rb

```diff
@@ -237,7 +237,7 @@ def test_batch(batch_id, env, running_total, tests)
       FileUtils.mkpath(test_batch_dir)
 
       test_state = "#{test_batch_dir}/test_state.json"
-      cmd = "#{@test_command} --format json --out #{test_state} #{@cucumber_options} "
+      cmd = "#{@test_command} --format ParallelCucumber::Helper::Cucumber::JsonStatusFormatter --out #{test_state} #{@cucumber_options} "
```

Comment:
`::ParallelCucumber::Helper::Cucumber::JsonStatusFormatter`

put formatter fqn to constant and use constant here and above